libs/coin-evm/docs/signOperation.md

```diff
@@ -0,0 +1,15 @@
+# `signOperation.ts`
```

Comment:
there is already doc in the code let's merge infos

Reply:
Same answer as above